financial-connections/src/test/java/com/stripe/android/financialconnections/FinancialConnectionsSheetTest.kt

```diff
@@ -17,6 +17,11 @@ class FinancialConnectionsSheetTest {
     @Test
     fun `present() should launch the connection sheet with the given configuration`() {
         financialConnectionsSheet.present(configuration)
-        verify(financialConnectionsSheetLauncher).present(configuration)
+        verify(financialConnectionsSheetLauncher).present(
+            FinancialConnectionsSheetConfiguration(
+                ApiKeyFixtures.DEFAULT_FINANCIAL_CONNECTIONS_SESSION_SECRET,
+                ApiKeyFixtures.DEFAULT_PUBLISHABLE_KEY
+            )
+        )
```

Comment:
Why can’t we use `configuration` anymore?

Reply:
we can't move `FinancialConnectionsSheet.Configuration` to `financial-connections-core` without introducing breaking changes - The best alternative we have is to introduce an internal class to use it within the SDK (kind of what we do today with exposed Results and internal `ActivityResults`).